src/main/java/edu/ucsb/cs156/gauchoride/entities/RideRequest.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "riderequests")
+public class RideRequest {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long userid;
+  private String day;
```

Comment:
It would be good to establish (at least with a comment) whether this is:
* a single letter M, T, W, R, F, S, U
* three letter abbrev Mon,Tue,Wed,Thu,Fri,Sat,Sun
* fully spelled out days of the week.

If you don't do this early, you get a mishmash of different programmers making different assumptions leading to bugs, bugs, bugs.